minecraft_servers/uroria/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "uroria",
+  "nice_name": "Uroria Network",
+  "direct_ip": "play.uroria.com",
+  "server_wildcards": [
+    "%.uroria.com",
+    "%.uroria.de",
+    "%.uroria.net"
+  ],
+  "supported_languages": [
+    "de",
+    "lt",
+    "en",
+    "fr",
+    "la"
+  ],
+  "social": {
+    "web": "https://uroria.com",
+    "instagram": "urorianetwork",
+    "discord": "https://discord.com/invite/RGX8rPshFG",
+    "twitter": "UroriaNetwork",
+    "youtube": "https://yt.uroria.com"
```

Comment:
Custom links are only supported for partner servers due to security reasons.

Reply:
Well I've removed them already